packages/integrations/components/integrationsManager/index.js

```diff
@@ -2,62 +2,52 @@ import React from 'react';
 import PropTypes from 'prop-types';
 // User Config.
 import userConfig from '@irvingjs/irving.config';
-// Redux.
-import { connect } from 'react-redux';
-import { actionHydrateComponents } from '../../actions';
 // Component Map.
 import componentMap from './componentMap';
 
 /**
  * Integrations Manager.
  *
  * A component that consumes a `irving/integrations` configuration from the
- * REST API, Irving config, or persisted Redux state slice and renders the
- * corresponding integrations on the client.
+ * REST API and renders the corresponding integrations on the client.
  *
  * @param {Object}   props - All props.
  * @param {Object}   props.integrations - Config object for active integrations.
- * @param {boolean}  props.isHydrated - Has the component map already been hydrated?
- * @param {function} props.hydrateComponents - Redux action for updating the `componentMap` in the state tree.
- * @param {Object}   props.hydratedComponents - The existing component key/prop pairs.
  */
-const IntegrationsManager = ({
-  integrations,
-  isHydrated,
-  hydrateComponents,
-  hydratedComponents,
-}) => {
-  // This check is run on server-side renders to ensure that the component hydration
-  // is ran a single time. After hydration, `isHydrated` is set to `true` and will
-  // only be re-run if the component props change during the component's lifecycle.
-  if (false === isHydrated) {
-    // Retrieve the keys available for render in the component map.
-    const keyMap = Object.keys(integrations);
+const IntegrationsManager = ({ integrations }) => {
```

Comment:
More of a pet peeve than something wrong here, but I generally prefer to destructure in the function body instead of params to allow for easier debugging and/or console logging. I think we've done this differently in different projects, but I'd prefer to standardize around not use destructuring in params.

Reply:
I don't have strong opinions about which pattern we use. This component was already destructuring in params, so I left it.